examples/website/geospatial/app.tsx

```diff
@@ -21,10 +21,11 @@ import {INITIAL_LOADER_NAME, INITIAL_EXAMPLE_NAME, INITIAL_MAP_STYLE, EXAMPLES}
 import {Table, GeoJSON} from '@loaders.gl/schema';
 import {Loader, load /* registerLoaders */} from '@loaders.gl/core';
 import {GeoArrowLoader} from '@loaders.gl/arrow';
-import {GeoParquetLoader, installBufferPolyfill} from '@loaders.gl/parquet';
+import {GeoParquetLoader, installBufferPolyfill, preloadCompressions} from '@loaders.gl/parquet';
 import {FlatGeobufLoader} from '@loaders.gl/flatgeobuf';
 import {ShapefileLoader} from '@loaders.gl/shapefile';
 import {KMLLoader, GPXLoader, TCXLoader} from '@loaders.gl/kml';
+import {ZstdCodec} from 'zstd-codec';
```

Comment:
See if you can still build the website. In my experience, the zstd-codec module has some subtle node dependencies that breaks both website and worker builds...

Reply:
I've fixed the website. The change is in `website/webpack-plugin.js`. It blocks "no `fs` module" errors from `zstd-codec`

Reply:
That's great. That might unblock some other things as well, thank you!